NitroxModel/Platforms/Store/Steam.cs

```diff
@@ -252,6 +267,10 @@ private static ProcessStartInfo CreateSteamGameStartInfo(string gameFilePath, st
             result.EnvironmentVariables.Add("WINEPREFIX", compatdataPath);
             result.EnvironmentVariables.Add("STEAM_COMPAT_CLIENT_INSTALL_PATH", steamPath);
             result.EnvironmentVariables.Add("STEAM_COMPAT_DATA_PATH", compatdataPath);
+            
+            // Enable Steam overlay and API for controller input and OSK support (Proton-specific)
+            // Note: SteamAppId and other overlay vars are already set in the initial EnvironmentVariables collection
```

Comment:
The note isn't needed and hard to keep in sync if code would change. Better to let a programmer ctrl+f for SteamAppId if they want to know if or where it's assigned.